hammers/scripts/enforce_retirement.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+
+import mysql.connector
+import os
+from hammers.slack import Slackbot
+from hammers.util import base_parser
+
+def correct_state(cursor,slk,dryrun=False):
+    # Find retired nodes
+    cursor.execute("SELECT n.uuid from ironic.nodes n join blazar.computehosts ch on n.uuid = ch.hypervisor_hostname WHERE n.name LIKE '%retired' AND ch.reservable != 0;")
+    retired_nodes = cursor.fetchall()
+    for node in retired_nodes:
+        if not dryrun:
+            blazar_fix = "UPDATE blazar.computehosts SET reservable = '0' WHERE hypervisor_hostname = %s"
+            cursor.execute(blazar_fix, [node[0]])
+            mess = ("Reverted state of node " + node[0] + " to non-reservable.")
+        else:
+            mess = ("State of retired node " + node[0] + " is reservable, run without '--dryrun' to retire.")
+        print(mess)
+        if slk:
+            slk.message(mess)
```

Comment:
I think this will be way less chatty if you construct the message as you're going, and then send out one bulk message.

```python
message_lines = []

for node in retired_nodes:
    # Do stuff...
    message_lines.append("Reverted state...")

if message_lines:
    message = '\n'.join(message_lines)
    print(message)
    if slk:
        slk.message(message)
```